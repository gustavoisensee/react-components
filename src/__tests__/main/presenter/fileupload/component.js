import React from 'react';
import renderer from 'react-test-renderer';
import PresenterFileUpload from './../../../../modules/main/presenter/fileupload/component';

describe('Main/presenter/fileupload component test', () => {
  test('Main/presenter/fileupload component renders', () => {
    const props = {
      items: [],
      upload: () => {},
      remove: () => {}
    };

    const tree = renderer.create(
      <PresenterFileUpload {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
