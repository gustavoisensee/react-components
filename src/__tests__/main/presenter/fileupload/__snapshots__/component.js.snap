exports[`Main/presenter/fileupload component test Main/presenter/fileupload component renders 1`] = `
<div
  className="content">
  <div
    className="title">
    <span>
      FileUpload
    </span>
  </div>
  <div
    className="example">
    <div
      className="content">
      <div
        className="content__input btn">
        <input
          className="uploader"
          id="186201714551"
          multiple={true}
          onChange={[Function]}
          type="file" />
        <label
          className="uploader__title"
          htmlFor="186201714551">
          Select file
        </label>
      </div>
      <div
        className="content__items" />
    </div>
  </div>
  <div
    className="description">
    Description
  </div>
</div>
`;
